src/main/java/uk/gov/pay/adminusers/service/UserServices.java

```diff
@@ -175,6 +174,7 @@ public UserServices(UserDao userDao,
     @Transactional
     public Optional<User> authenticateSecondFactor(String externalId, int code) {
         logger.debug("OTP attempt - user_id={}", externalId);
+        ZonedDateTime now = ZonedDateTime.now(ZoneId.of("UTC"));
```

Comment:
You could also use `ZonedDateTime.now(ZoneOffset.UTC)`

Reply:
Yes, looks like we use `ZonedDateTime.now(ZoneId.of("UTC"))` a bit more in the project